tests/trezor/trezor_tests.cpp

```diff
@@ -393,6 +462,20 @@ static device_trezor_test *ensure_trezor_test_device(){
   return trezor_device;
 }
 
+static size_t min_mixin_for_hf(uint8_t hf)
+{
+  if (hf >= HF_VERSION_MIN_MIXIN_15)
+    return 15;
+  if (hf >= 8)
+    return 10;
+  return 0;
+}
```

Comment:
From what I found in `blockchain.cpp`: https://github.com/monero-project/monero/blob/master/src/cryptonote_core/blockchain.cpp#L3337
It should be:
```
  if (hf >= 2)
    return hf >= HF_VERSION_MIN_MIXIN_15 ? 15 : hf >= HF_VERSION_MIN_MIXIN_10 ? 10 : hf >= HF_VERSION_MIN_MIXIN_6 ? 6 : hf >= HF_VERSION_MIN_MIXIN_4 ? 4 : 2;

  return 0;

Reply:
Our Trezor integration does not support HF below 8 (i.e., `HF_VERSION_MIN_MIXIN_10`) so we are not using it also in our test code. Is that ok?